src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpRequestStream.cs

```diff
@@ -110,7 +110,7 @@ public override Task WriteAsync(byte[] buffer, int offset, int count, Cancellati
 
             if (count > buffer.Length - offset)
             {
-                throw new ArgumentException(nameof(buffer));
+                throw new ArgumentException(SR.Argument_BufferNotLongEnough, nameof(buffer));
```

Comment:
You must copy string resources into both strings.resx files when changing WinHttpHandler.  Strings in WinHttpHandler strings.resx must be copied into Http strings.resx library. This is because Http library builds the WinHttpHandler source code directly inside it.